src/agent/LibFuzzerDotnetLoader/Program.cs

```diff
@@ -81,4 +75,63 @@ static bool TryTestOne<T>(MethodInfo method, Func<T, SharpFuzz.ReadOnlySpanActio
             return TryTestOne<TestOneArray>(method, t => span => t(span.ToArray()));
         }
     }
+
+    class LibFuzzerDotnetTarget
+    {
+        public string AssemblyPath { get; }
+        public string ClassName { get; }
+        public string MethodName { get; }
+
+        public LibFuzzerDotnetTarget(string assemblyPath, string className, string methodName)
+        {
+            AssemblyPath = assemblyPath;
+            ClassName = className;
+            MethodName = methodName;
+        }
+
+        public static LibFuzzerDotnetTarget FromEnvironment()
+        {
+            try {
+                return FromEnvironmentVarDelimited();
+            }
+            catch
+            {}
+
+            try {
+                return FromEnvironmentVars();
+            }
+            catch
+            {}
+
+            throw new Exception("No fuzzing target specified by environment variables");
+        }
+
+        static LibFuzzerDotnetTarget FromEnvironmentVars()
+        {
+            var assemblyPath = Environment.GetEnvironmentVariable(EnvVar.ASSEMBLY)??
+                throw new Exception($"`{EnvVar.ASSEMBLY}` not set");
+            var className = Environment.GetEnvironmentVariable(EnvVar.CLASS)??
+                throw new Exception($"`{EnvVar.CLASS}` not set");
+            var methodName = Environment.GetEnvironmentVariable(EnvVar.METHOD)??
+                throw new Exception($"`{EnvVar.METHOD}` not set");
+
+            return new LibFuzzerDotnetTarget(assemblyPath, className, methodName);
+        }
+
+
+        static LibFuzzerDotnetTarget FromEnvironmentVarDelimited()
+        {
+            var target = Environment.GetEnvironmentVariable(EnvVar.TARGET)??
+                throw new Exception($"`{EnvVar.TARGET}` not set.");
+
+            var parts = target.Split(':', StringSplitOptions.RemoveEmptyEntries);
+
+            if (parts.Length < 3)
+            {
+                throw new Exception($"Value of `{EnvVar.TARGET}` is invalid");
```

Comment:
```suggestion
                throw new Exception($"Value of `{EnvVar.TARGET}` is invalid, should be of the form: '{{Assembly Path}}:{{Class Name}}:{{Method Name}}'");
```

Reply:
Ha, this is basically what I had originally! I'll add it back in, even though we always catch this exception (for now).